client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -2430,18 +2433,29 @@ Undocumented.prototype.checkSiteAddressValidation = function( siteId, siteAddres
  *
  * @param {int} [siteId] The siteId for which to change the address
  * @param {object} [blogname]	The desired new site address
+ * @param {string} [domain] The domain name of the new site address (ex. news.blog, wordpress.com, etc.)
+ * @param {string} [oldDomain] The full domain name of the original site (ex. mysite.news.blog, mysite.wordpress.com, etc.)
```

Comment:
Super minor, and that's probably because I don't know this area of our code, but I find confusing that `domain` and `oldDomain` refer to slightly different things.